contracts/examples/guards/DebugTransactionGuard.sol

```diff
@@ -85,4 +85,20 @@ contract DebugTransactionGuard is BaseGuard {
         txNonces[txHash] = 0;
         emit GasUsage(msg.sender, txHash, nonce, success);
     }
+
+    /**
+     * @notice Called by the Safe contract before a transaction is executed via a module.
+     * @param to Destination address of Safe transaction.
+     * @param value Ether value of Safe transaction.
+     * @param data Data payload of Safe transaction.
+     * @param operation Operation type of Safe transaction.
+     * @param msgSender Account executing the transaction.
+     */
+    function checkModuleTransaction(
```

Comment:
This guard will not work, right? Should we update this example into a working state?

Reply:
Updated